gm4_scuba_gear/data/gm4_scuba_gear/functions/recipe_check.mcfunction

```diff
@@ -1,4 +1,4 @@
+execute if score @s gm4_slot_count matches 9 if score @s gm4_stack_size matches 1 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:gold_ingot"},{Slot:1b,id:"minecraft:potion",tag:{Potion:"minecraft:long_water_breathing"}},{Slot:2b,id:"minecraft:gold_ingot"},{Slot:3b,id:"minecraft:scute"},{Slot:4b,id:"minecraft:gold_ingot"},{Slot:5b,id:"minecraft:scute"},{Slot:6b,id:"minecraft:gold_ingot"},{Slot:7b,id:"minecraft:gold_ingot"},{Slot:8b,id:"minecraft:gold_ingot"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"golden_chestplate",Count:1b,tag:{CustomModelData:1,gm4_scuba_gear:{item:"tank"},display:{Name:'{"translate":"%1$s%3427655$s","with":["SCUBA Tank",{"translate":"item.gm4.scuba_tank"}],"italic":false}'},AttributeModifiers:[{AttributeName:"generic.movement_speed",Name:"generic.movement_speed",Amount:-.1,Operation:1,UUID:[I;0,911382,0,189444],Slot:"chest"},{AttributeName:"generic.armor",Name:"generic.armor",Amount:4,Operation:0,UUID:[I;0,847530,0,778609],Slot:"chest"}]}}]}
+execute if score @s gm4_slot_count matches 6 if score @s gm4_stack_size matches 1 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:scute"},{Slot:2b,id:"minecraft:scute"},{Slot:3b,id:"minecraft:rabbit_hide"},{Slot:5b,id:"minecraft:rabbit_hide"},{Slot:6b,id:"minecraft:rabbit_hide"},{Slot:8b,id:"minecraft:rabbit_hide"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"leather_boots",Count:1b,tag:{HideFlags:64,CustomModelData:2,gm4_scuba_gear:{item:"flippers"},display:{Name:'{"translate":"%1$s%3427655$s","with":["Flippers",{"translate":"item.gm4.flippers"}],"italic":false}',color:11009871},AttributeModifiers:[{AttributeName:"generic.armor",Name:"generic.armor",Amount:1,Operation:0,UUID:[I;0,881952,0,167143],Slot:"feet"},{AttributeName:"generic.movement_speed",Name:"generic.movement_speed",Amount:-.4,Operation:1,UUID:[I;0,579389,0,942428],Slot:"feet"}]}}]}
```

Comment:
I dislike that the item data is both hardcoded here and present in the loot-table, however CC is not currently setup to easily accept items created by a loottable, so I suppose it is alright. Food for thought I suppose.